### PR TITLE
Simplify UX to Enable VCP - Docker Image for Manager Pod and Daemonset Pod

### PR DESCRIPTION
**Pull request by divyenpatel**
Issue: https://github.com/vmware/kubernetes/issues/210

cc: @SandeepPissay @BaluDontu @tusharnt

### CODE REVIEW

File: enable-vcp-image/install.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+export GOROOT="/opt/go"
+export GOPATH="/root/.go"
+
+cd /opt
+curl -LO https://storage.googleapis.com/golang/go${GOVERSION}.linux-amd64.tar.gz
+tar zxf go${GOVERSION}.linux-amd64.tar.gz && rm go${GOVERSION}.linux-amd64.tar.gz
+ln -s /opt/go/bin/go /usr/bin/
+
+mkdir $GOPATH
+go get github.com/vmware/govmomi/govc
```
Comment (SandeepPissay) [inline enable-vcp-image/install.sh]: Why are we getting the govc code? Shouldn't we just download the latest release?

File: enable-vcp-image/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:16.10
```
Comment (SandeepPissay) [inline enable-vcp-image/Dockerfile]: A general comment on the location where we are placing these files. Dont we want to place it under vsphere package in kubernetes repo? This way the open source community can also maintain the scripts.

File: enable-vcp-image/install.sh (hunk 1 above)
Comment (divyenpatel) [inline enable-vcp-image/install.sh]: We can install the latest govc version from source if we have the Go toolchain installed using `go get`
https://github.com/vmware/govmomi/tree/master/govc#source

Do you want to find the latest govc release version and build URL and use the CURL?
This looks easier and straightforward to me.

File: enable-vcp-image/Dockerfile (hunk 2 above)
Comment (divyenpatel) [inline enable-vcp-image/Dockerfile]: We cannot put this kind of scripts in the vsphere package on Kubernetes Repository. It is restricted to Cloud Provider code and Volume plug-in code.

File: enable-vcp-image/install.sh (hunk 1 above)
Comment (divyenpatel) [inline enable-vcp-image/install.sh]: As discussed offline used `curl` to download per-built  binary

**Pull request merged**